generated/user_existence/user_existence.json

```diff
@@ -0,0 +1,29 @@
+{
+  "primary_applicant": {
+    "id": "58bf3bb6e41b0738bf0000ce",
+    "user_id": "58bf3b87e41b0738bf0000cc",
```

Comment:
id -> person ID
user_id -> user ID

Given that we didn't necessarily define or decide on a concrete list of fields to expose, this was my first, _pretty random_, stab at what may or may not be useful in addition to the ones you mentioned on Slack.